Formula/libzim.rb

```diff
@@ -0,0 +1,41 @@
+class Libzim < Formula
+  desc "Reference implementation of the ZIM specification"
+  homepage "https://github.com/openzim/libzim"
+  url "https://download.openzim.org/release/libzim/libzim-8.1.0.tar.xz"
+  sha256 "8c9bda942772bb8de1acf4832d4bda5c913d9595506187b62a35ae15e530221f"
+  license "GPL-2.0-only"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "icu4c"
+  depends_on "xapian"
+  depends_on "xz"
+  depends_on "zstd"
+
+  uses_from_macos "python" => :build
+
+  def install
+    mkdir "build" do
+      system "meson", *std_meson_args, ".."
+      system "ninja", "-v"
+      system "ninja", "install", "-v"
+    end
+  end
+
+  test do
+    (testpath/"test.cpp").write <<~EOS
+      #include <iostream>
+      #include <zim/version.h>
+      int main(void) {
+        zim::printVersions(); // first line should print "libzim <version>"
+        return 0;
+      }
+    EOS
+    system ENV.cxx, "test.cpp", "-L#{lib}", "-lzim", "-o", "test", "-std=c++11"
```

Comment:
Doesn't this work without `-std`?

Reply:
No, the header parsing fails without `-std=c+11`